kalite/testing/mixins/browser_mixins.py

```diff
@@ -316,25 +324,31 @@ def browser_login_user(self, username, password, facility_name=None, browser=Non
         """
         browser = browser or self.browser
 
-        login_url = self.reverse("login")
+        login_url = self.reverse("homepage")
```

Comment:
I did the minimal change to get it working. Very happy to make it use the endpoint instead.